recipes/js_regex/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: js-regex
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/js-regex/js-regex-{{ version }}.tar.gz
+  sha256: 90bbc2d2b76c2d9b07058e59e089fb175c7eccb8f82f55bea5bdb3515c03ed8d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - js_regex
+
+about:
+  home: https://github.com/Zac-HD/js-regex
+  license: MPL-2.0
+  license_family: MOZILLA
+  license_file: LICENSE
```

Comment:
Can you open an issue or send a PR upstream for the missing license file in sdist?

Reply:
Sent a PR: https://github.com/Zac-HD/js-regex/pull/5